lib/LDAPGroupManager.php

```diff
@@ -82,15 +92,27 @@ public function respondToActions() {
 	 * @return string|null
 	 */
 	public function createGroup($gid) {
+		$adminUser = $this->userSession->getUser();
+		$requireActorFromLDAP = $this->configuration->isLdapActorRequired();
+		if ($requireActorFromLDAP && !$adminUser instanceof IUser) {
+			throw new Exception('Acting user is not from LDAP');
+		}
+		try {
+			$connection = $this->ldapProvider->getLDAPConnection($adminUser->getUID());
+			// TODO: what about multiple bases?
+			$base = $this->ldapProvider->getLDAPBaseGroups($adminUser->getUID());
```

Comment:
```suggestion
			$base = $this->ldapProvider->getLDAPBaseGroups($adminUser->getUID())[0];
```
To match line 112, no?